src/integrations/expressApollo.ts

```diff
@@ -112,10 +112,6 @@ export function graphqlHTTP(options: ExpressApolloOptions | ExpressApolloOptions
           params = optionsObject.formatParams(params);
         }
 
-        if (!params.query) {
```

Comment:
Oh, hm. What was the error exactly? What request was GraphiQL making that caused this error to be thrown? surely the introspection query is sent with a query body, no?